scripts/runWorker.pl

```diff
@@ -56,7 +56,7 @@ sub main {
                'analysis_id=i'              => \$analysis_id,
                'logic_name=s'               => \$logic_name,
                'job_id=i'                   => \$job_id,
-               'force=i'                    => \$force,
+               'force'                      => \$force,
```

Comment:
You'll have to change `-force 1` in docs/running_pipelines/running.rst

Reply:
There are also several `-force 1` in docs/running_pipelines/troubleshooting.rst
BTW [troubleshooting.rst](http://ensembl-hive.readthedocs.io/en/master/running_pipelines/troubleshooting.html#the-runworker-pl-script) overlaps [running.rst](http://ensembl-hive.readthedocs.io/en/master/running_pipelines/running.html#running-a-single-job-using-runworker-pl) quite a lot